src/Sylius/Component/Core/Model/ImageOwnerInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Model;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface ImageOwnerInterface
```

Comment:
From my POV introducing `ImageOwnerInterface` does not bring us any real benefits, but only complicates things. It gives me nothing to know that `Image::getOwner()` returns an object implementing `ImageOwnerInterface` because I can't do anything with that knowledge.